config.rb

```diff
@@ -110,7 +110,7 @@
   content.gsub(%r{^<(h[0-6])(.*?)</\1>}m) do |header_text|
     header = Nokogiri::HTML::DocumentFragment.parse(header_text).children.first
     id = header.attr(:id)
-    header.children.before("<a class='anchor' href='##{id}'></a>") if id
+    # header.children.before("<a class='anchor' href='##{id}'></a>") if id // Commenting this out, as it appears to serve no function and is generating empty links, which causes accessibility issues. 
```

Comment:
Thanks, @Awjin. The impact is that it is generating empty links, which create confusion for screenreader users. Perhaps we can visually hide the link text using a `<span class="visuallyhidden">` like this:
```
     header.children.before("<a class='anchor' href='##{id}'><span class='visuallyhidden'>{id}</span></a>") if id 
```
Does that work for you?

Reply:
That works for me.

Just double-checking, the span should be `<span class='visuallyhidden'>`**#{id}**`</span></a>`.

What's the best practice for accessible text on links? Is it ok that that will generate hyphenated text ("loading-members")?

Reply:
Appreciate the support!

Ah, yeah. Ideally, it would be human readable, so it's preferable that it's "Loading members." Is it possible to either generate the name from the existing header (which is already written properly) or strip out the punctuation with a script? Otherwise the syntax is correct.

Reply:
`#{header.text}` should give you human readable text